tools/ci/stage.sh

```diff
@@ -48,11 +52,23 @@ flink-metrics,\
 flink-metrics/flink-metrics-core,\
 flink-external-resources,\
 flink-external-resources/flink-external-resource-gpu,\
+flink-libraries,\
 flink-libraries/flink-cep,\
 flink-libraries/flink-cep-scala,\
-flink-libraries/flink-state-processing-api"
+flink-libraries/flink-state-processing-api,\
+flink-libraries/flink-gelly,\
+flink-libraries/flink-gelly-scala,\
+flink-libraries/flink-gelly-examples,\
+flink-queryable-state,\
+flink-queryable-state/flink-queryable-state-runtime,\
+flink-queryable-state/flink-queryable-state-client-java,\
+flink-container,\
+flink-dstl,\
+flink-dstl/flink-dstl-dfs,\
```

Comment:
I see, I should have noticed the hint with `zsh: event not found: ,!`. But thanks for clarification. I missed the `\` in the negation. I interpreted it as a backspace for regex stuff... The backspace doesn't appear in the AzureCI output because `echo` will omit the escaping character...